File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/DocumentFieldCreatePhotoValue.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param url Photo URL for a JPG image
 */
data class DocumentFieldCreatePhotoValue (

        @JsonProperty("url") val url: String? = null
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/DocumentFieldCreatePhotoValue.scala

package org.openapitools.client.model


case class DocumentFieldCreatePhotoValue (
    /* Photo URL for a JPG image */
    _url: Option[String]
)
object DocumentFieldCreatePhotoValue {
    def toStringBody(var_url: Object) =
        s"""
        | {
        | "url":$var_url
        | }
        """.stripMargin
}
